meson.build

```diff
@@ -0,0 +1,55 @@
+project('xtensor', 'cpp',
+        version:'0.21.5',
+        license:'BSD-3-Clause',
+        default_options : ['warning_level=3', 'cpp_std=c++14'])
+
+xtl_dep = dependency('xtl',
+    version:['>=0.6.12', '<1.0.0'],
+    fallback : ['xtl', 'xtl_dep'],
```

Comment:
Likewise here, if https://github.com/mesonbuild/quantstack-xtl/pull/1 includes a provide section, you should not need an explicit fallback.